### ISSUE TITLE
How to use with sentinel?

### ISSUE THREAD

**Issue by richmolj** [ember-cli-deploy/ember-cli-deploy-redis#48]
I've been attempting to pass my own `redisClient`, so I could use sentinel-aware `ioredis`. It looks like this won't work, and the injection is only for testing, since `redisClient` needs decorations provided by this repo (`upload`, `activate`, etc) and it doesn't seem possible to use an alternate redis lib ([since it relies on a particular method signature](https://github.com/ember-cli-deploy/ember-cli-deploy-redis/blob/master/lib/redis.js#L26-L30)). Am I understanding this correctly? Anything I can do here?

**Comment by lukemelia** [comment]
@richmolj yes, the redisClient config was created in the context of testing. You could provide your own wrapper around the ioredis, though, which adapts the required method signature appropriately.

**Comment by ghedamat** [comment]
@richmolj was luke's suggestion enough? or were you required to make other changes for this to work?
